app/templates/main/volunteerProfile.html

```diff
@@ -56,13 +43,17 @@ <h6 align="center">{{volunteer.email}}</h6>
            </tr>
        {% endfor %}
      {% else %}
-       <h4><p class="text-dark">You have no upcoming events. </p></h4>
-       <h4><p class="text-dark">Choose interests below to be notified about your upcoming events.</p><h4>
+     <div class="row">
+       <h4 class="col-sm-10"><p>Upcoming Events: </p></h4>
+       <a class="col-sm-2 nav-link float-end" href="/profile/{{volunteer.username}}/serviceTranscript">View Service Transcript</a>
+     </div>
+       <h5><p>You have no upcoming events.</p></h5>
      {% endif %}
    </table>
  </div>
  <!-- Start Program Eligibility Table -->
  <div class="row table-responsive">
+  <caption><h5>Choose interests below to be notified about your upcoming events:</h5></caption>
```

Comment:
what does the caption tag do? This isn't a caption.

Reply:
resolved